crates/router/src/connector/volt.rs

```diff
@@ -27,8 +33,18 @@ use crate::{
     utils::BytesExt,
 };
 
-#[derive(Debug, Clone)]
-pub struct Volt;
+#[derive(Clone)]
+pub struct Volt {
+    amount_converter: &'static (dyn AmountConvertor<Output = MinorUnit> + Sync),
```

Comment:
There's no need of dynamic dispatch here, As I can see below you are initialising it to `MinorUnitForConnector`. 
Could be just 
```rust
struct Volt(MinorUnitForConnector);
```